src/tools/rustfmt/ci/integration.sh

```diff
@@ -2,7 +2,7 @@
 
 set -ex
 
-: ${INTEGRATION?"The INTEGRATION environment variable must be set."}
+: "${INTEGRATION?'The INTEGRATION environment variable must be set.'}"
```

Comment:
In shell, simple quote strings are literals, whereas double quote strings are an equivalent of `format!`.

So now, variables (`${}`) are interpreted as strings, so if they contain whitespace characters, it might break the script. Hence why we're putting them in double quote strings so they are treated as one string.